### PR TITLE
Experiment: temperature sensor

### PR DESCRIPTION
**Pull request by minacode**
This PR is a test-implementation for #1327 .
I added the temperature value from the driver to the digital watch face.
And the value is: (drum roll...) 1. Always.

If someone sees something I did not see to make this useful, please send help!

I am adding this here as a Draft, because it might be of interest for some people, but is obviously not something we want to merge.

### CODE REVIEW

**Comment by ght** [comment]
Thanks for investigating! I was wondering about this too since finding out about it in the datasheet. Kinda disappointing that it does not seem to work.

**Comment by minacode** [comment]
[Here](https://github.com/BoschSensortec/BMA423-Sensor-API/blob/master/examples/generic/temperature/temperature.c) is an example that might be useful.

File: src/displayapp/screens/WatchFaceDigital.h [hunk 1]
```diff
@@ -51,19 +51,21 @@ namespace Pinetime {
...
         DirtyValue<std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds>> currentDateTime {};
         DirtyValue<bool> motionSensorOk {};
-        DirtyValue<uint32_t> stepCount {};
-        DirtyValue<uint8_t> heartbeat {};
-        DirtyValue<bool> heartbeatRunning {};
+        // DirtyValue<uint32_t> stepCount {};
+        // DirtyValue<uint8_t> heartbeat {};
+        // DirtyValue<bool> heartbeatRunning {};
         DirtyValue<bool> notificationState {};
+        DirtyValue<bool> temperature {};
```
Comment (ght) [inline src/displayapp/screens/WatchFaceDigital.h:58]:
```suggestion
        DirtyValue<uint32_t> temperature {};
```

**Review by ght — CHANGES_REQUESTED** [review]
I just tested this branch and it actually works fine if you fix the type!

**Comment by minacode** [comment]
Good catch, thank you!

**Comment by minacode** [comment]
Now its always 32000. Do you have useful values?

**Comment by ght** [comment]
Yes, but it changes very slowly. Try leaving it in the fridge for 5 minutes.

**Comment by minacode** [comment]
So I am getting values between 25 and 32 (times 1000). They match the surroundings a little bit, but they change very slowly. 
But I don't think that they are sensitive enough for a temperature app. I will continue the measurements for a while though.

**Comment by minacode** [comment]
I am closing this one because I sadly don't see any future for this.

**Pull request closed without merging**